docs/examples/divider/vertical-divider.vue

```diff
@@ -3,7 +3,7 @@
     <span>Rain</span>
     <el-divider direction="vertical"></el-divider>
     <span>Home</span>
-    <el-divider direction="vertical"></el-divider>
+    <el-divider direction="vertical" line-style="dashed"></el-divider>
```

Comment:
`border-style` vs `lint-style`? @JeremyWuuuuu WDYT

Reply:
in my opinion, border have four lines, and divider just have one line.

Reply:
I'd prefer `border-style`